.github/workflows/release.yml

```diff
@@ -27,13 +27,11 @@ jobs:
       - name: Launch release process
         run: |
           make release-build-platform PLATFORM=amd64
-          make release-build-platform PLATFORM=arm64
           make release-sha
-          make release-git
 
-      - name: Upload release folder
-        uses: actions/upload-artifact@v2
-        if: github.event.release
+      - name: attach binaries
+        uses: softprops/action-gh-release@v1
         with:
-          name: release
-          path: release
+          files: release/*
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```Error: ⚠️ GitHub Releases requires a tag```

Do we need this token? The GH action does not seem to be working

Reply:
issue is a bit different, but yes action requires token for uploading files to release

Reply:
It is strange that this action was triggered, since we cannot always tag something in a PR.